files/KoreBuild/scripts/dotnet-install.ps1

```diff
@@ -501,89 +498,109 @@ function Prepend-Sdk-InstallRoot-To-Path([string]$InstallRoot, [string]$BinFolde
     }
 }
 
-$CLIArchitecture = Get-CLIArchitecture-From-Architecture $Architecture
-$SpecificVersion = Get-Specific-Version-From-Version -AzureFeed $AzureFeed -Channel $Channel -Version $Version
-$DownloadLink = Get-Download-Link -AzureFeed $AzureFeed -SpecificVersion $SpecificVersion -CLIArchitecture $CLIArchitecture
-$LegacyDownloadLink = Get-LegacyDownload-Link -AzureFeed $AzureFeed -SpecificVersion $SpecificVersion -CLIArchitecture $CLIArchitecture
+$lockFile = Join-Path $PSScriptRoot "dotnetinstall.lock"
```

Comment:
Putting this lockfile next to the dotnet-install.sh script doesn't solve the problem if two different copies of this script are invoked in parallel. What if the script placed the lockfile in the target install directory?